tests/testthat/test-extractors.R

```diff
@@ -3,8 +3,8 @@ test_that("tweets_data works", {
 
   ## get data on most recent tweet from user(s)
   tweets <- tweets_data(jack)
-  expect_s3_class(tweets, "tbl")
-  expect_equal(colnames(tweets), tweets_names())
+  expect_s3_class(tweets, "tbl_df")
+  expect_true("status_id" %in% names(tweets))
```

Comment:
Why make the test less complete?

Reply:
Because requesting information about a user no longer returns all the data about their last tweet.